ExtensionExamples/ActionExtension/DocumentAction.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Intelledox.Extension.Action;
+using Intelledox.QAWizard;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace SampleActionExtensions
+{
+    /// <summary>
+    /// This sample demonstrates dealing within generated documents that have been sent into the action.
```

Comment:
Re-worded the description. Left the second part. Not sure if we should be including Design level setup in samples though. Doesn't hurt in this case.